bibs/Bad_Nauheim.json

```diff
@@ -11,6 +11,20 @@
     "country": "Deutschland",
     "data": {
         "baseurl": "http://bad-nauheim.bibdia-hosts.de",
+        "mediatypes": {
+            "MTYP0": "ALL",
+            "MTYP1": "BOOK",
+            "MTYP2": "MAGAZINE",
+            "MTYP3": "CD_SOFTWARE",
+            "MTYP4": "MOVIE",
+            "MTYP5": "AUDIO_CASSETTE",
+            "MTYP6": "CD_MUSIC",
+            "MTYP9": "BOARDGAME",
+            "MTYP10": "MAP",
+            "MTYP11": "DVD",
+            "MTYP24": "BLURAY",
+            "MTYP26": "GAME_CONSOLE"
+        },
```

Comment:
The "problem" is, that they are not using the icons on their [page](https://bad-nauheim.bibdia-hosts.de/opac/de/qsel_frm.html.S), rather they use images, e.g. https://bad-nauheim.bibdia-hosts.de/opac/image/all.gif.S.
`MTYP0` is the actual value of the input that is sent with the POST-request when `All` is selected as mediatype.

```html
<form name="qsel" onsubmit="javascript:smFrm2('de')" action="" method="POST">
 ...
 <table width="*" cellspacing="0" cellpadding="0" border="3" align="left">
  <tbody><tr>
   ...
  </tr>
 </tbody></table>
 <br><br><br>

 <table class="tab1" width="*" cellspacing="0" cellpadding="0" border="0">
  <tbody><tr>
    ...
    <input type="radio" name="MT" checked="" value="MTYP0"><img src="../image/all.gif.S" alt="Alles">
    <input type="radio" name="MT" value="MTYP1"><img src="../image/book.gif.S" alt="Buch" title="Buch">
    <input type="radio" name="MT" value="MTYP2"><img src="../image/paper.gif.S" alt="Zeitschrift">
    <input type="radio" name="MT" value="MTYP3"><img src="../image/cdrom.gif.S" alt="CD-ROM">
    <input type="radio" name="MT" value="MTYP4"><img src="../image/video.gif.S" alt="Video">
    <input type="radio" name="MT" value="MTYP5"><img src="../image/cass.gif.S" alt="Kassette">
    <input type="radio" name="MT" value="MTYP6"><img src="../image/cd.gif.S" alt="CD">
    <input type="radio" name="MT" value="MTYP9"><img src="../image/game.gif.S" alt="Spiele">
    <input type="radio" name="MT" value="MTYP10"><img src="../image/map.gif.S" alt="Karten">
    <input type="radio" name="MT" value="MTYP11"><img src="../image/dvd.gif.S" alt="DVDs">
    <input type="radio" name="MT" value="MTYP24"><img src="../image/bluray.gif.S" alt="Blue-Ray"> 
    <input type="radio" name="MT" value="MTYP26"><img src="../image/console.gif.S" alt="Konsolenspiele">    
   </td>
  </tr>
 </tbody></table>
 <br>
 ...
</form>
```
Will change to the image name instead of the input value.

Reply:
The `mediatypes` configuration parameter is only used for assigning media types the list of search results or lent/reserved items, not for parsing the search form! See
https://github.com/opacapp/opacclient/blob/master/opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BiBer1992.java#L451 and
https://github.com/opacapp/opacclient/blob/master/opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BiBer1992.java#L1078
https://github.com/opacapp/opacclient/blob/master/opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/BiBer1992.java#L1223
In the search form, we just use the `alt` tag of each item to get the correct label for the media type. So if search is not working in this library, this is not the problem.

Reply:
Removed mediatypes